curriculum/challenges/english/25-front-end-development/quiz-javascript-events/66edd0ac31fea6e678eb925a.md

```diff
@@ -17,439 +17,468 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which method is used to listen to events in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+`.getElementById()`
 
 ---
 
-Placeholder distractor 2
+`.createElement()`
 
 ---
 
-Placeholder distractor 3
+`.innerHTML`
 
 #### --answer--
 
-Placeholder answer
+`.addEventListener()`
+
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Following code is an example of?
+```js
+const parentList = document.getElementById('parent-list');
+
+parentList.addEventListener('click', function(event) {
+  if (event.target && event.target.nodeName === 'LI') {
+    console.log('List item clicked:', event.target.textContent);
+  }
+});
+```
```

Comment:
````suggestion
Following code is an example of?

```js
const parentList = document.getElementById('parent-list');

parentList.addEventListener('click', function(event) {
  if (event.target && event.target.nodeName === 'LI') {
    console.log('List item clicked:', event.target.textContent);
  }
});
```
````

code fences requires to be surrounded by empty lines